### DISCUSSION TITLE
connections to interfaces

### DISCUSSION THREAD

**Discussion by lasag-2511** [netbox-community/netbox#10145] [Q&A]
Hi all,

I am trying to add Interface connections on Netbox. for ex: E0 interface to eth0. Unfortunately I cannot do it. When i select interfaces and choose interface again.. I always get connect to cable. But there is nothing to connect to the inferfaces. Also on the interface connections, I cannot see add option. Only export option is available? How can i add interface connections ?

**Comment by a084ed22** [comment]
interfaces should be added on the device via add components > interfaces

**Reply by lasag-2511** [reply]
I have added interfaces via components., But i wanted to connect to eth0 But its not possible. for ex:  E0 is interface i wanted to connect to eth0. Under connections, i selected interfaces, but rather than connect to interface, it takes me to connect to cable.

**Reply by candlerb** [reply]
What version of Netbox are you using?

Cables *are* the way to connect interfaces together.  In summary:

* Go to device 1
* Select the Interfaces tab
* Along the row for the interface you want to connect, there's a plug-shaped drop-down menu
* Here you can select either "Interface", "Front Port", "Rear Port", or "Circuit Termination".  This is the type of thing you want to connect *to* at the other end.  If it's an interface on another device, then select "Interface"; if it's a patch panel then select "Front Port".
* Then you get the "create cable" page, with the A end already filled in.  Select the details on the B end, for the second device/interface that you want to connect to.

That's it.

If that's not what you're asking, can you you explain in more detail what you mean by "interface connections, for ex: E0 interface to eth0" ?

**Reply by lasag-2511** [reply]
Thanks for the reply. I found the way. I configured interface only on A side, But i need to have the interface connection on the B side. So we need to add interfaces on both A side and B side and create a connection.

**Reply by candlerb** [reply]
That's it - you can only connect to an interface that already exists.

**Reply by lasag-2511** [reply]
any idea how can we choose both front and rear rackspaces when defining a device?

**Reply by candlerb** [reply]
Click "full depth" in the device type

**Reply by lasag-2511** [reply]
Thanks for the quick reply.
